macros/find_all_hard_coded_references.sql

```diff
@@ -164,48 +164,48 @@
                 (?<!distinct\s)
 
                 # first matching group
-                # from or join followed by at least 1 whitespace character 
+                # from or join followed by at least 1 whitespace character
                 (from|join)\s+
 
                 # second matching group
-                # 1 or 0 of (opening bracket, backtick, or quotation mark)            
+                # 1 or 0 of (opening bracket, backtick, or quotation mark)
                 ([\[`\"\']?)
 
                 # third matching group
                 # at least 1 word character
-                (\w+)
+                (\w+-?\w+)
 
                 # fouth matching group
-                # 1 or 0 of (closing bracket, backtick, or quotation mark)            
+                # 1 or 0 of (closing bracket, backtick, or quotation mark)
                 ([\]`\"\']?)
 
                 # fifth matching group
-                # a period            
+                # a period
                 (\.)
 
                 # sixth matching group
                 # 1 or 0 of (opening bracket, backtick, or quotation mark)
                 ([\[`\"\']?)
 
                 # seventh matching group
-                # at least 1 word character            
-                (\w+)
+                # at least 1 word character
```

Comment:
wouldn't this change this to "at least 1 word character followed by 0 or 1 dash followed by at least 1 word character?)